index.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>FIRST CAPSTONE.</title>
+</head>
+<body class="full-body">
+    <img src="./images/Union.png" alt="hamburger-button" class="hamburger" id="openpopup">
+    
+    <div class="popup" id="popup">
+        <img src="./images/cloasepopup.svg" alt="close button" class="closepopup" id="closepopup">
+        <ul>
+           <li class="list" ><a href="about.html">About</a></li> 
+            <li class="list"><a href="index.html">Home</a></li>
+            <li class="list"><a href="#form">Join</a></li>
+            <li class="list"><a href="#contact">Contact us</a></li>
+        </ul>
+       
+    </div>
+
+    <div>
+      <a class ="nav-image" href="./about.html">
+      <img src="./images/240957a3-789c-4e2b-9fc7-34a08b3e9f0c.png" class="crypto-image" alt="crypto-logo" >
+      </a>
+    </div>
+
+    <nav class="navigation">
+       
+        <div class="nav-text">
+        <div class="about"> <a href="./about.html"> About</a></div>
+        <div class="home"> <a href="./index.html">Home</a></div>
+        <div class="join"><a href="#form">Join</a></div>
+        <div class="contact" > <a href="#contact">Contact us </a></div>
+        </div>
+    </nav>
+  
+
+    <section class="intro-section">
+
+      <article class="intro-align">
+    <div class="crypto-text" id="intro">"Blockchain crypto!"</div>
+
+    
+    <h1 class="ico-title">NEW CRYPTO COIN 2022 INITIAL COIN OFFERING</h1>
+    <div class="ico-words"> The crypto currency that has come to create financial freedom and be the foundation of the future of finance,all over the world. The ico will be taking place on December .</div>
+    <h2 class="ico-date">2022.12.01(TUE)~30(FRI)</h2>
+    <div id="contact" class="contact">@ www.fakenewcryptocoin.com</div>
+      </article>
+
+    </section>
+
+
+    <section class="main-section">
+        
+    <h2 class="program-class">Main Program</h2>
```

Comment:
Kindly remove the extra space in the desktop version.
![image](https://user-images.githubusercontent.com/61361037/219807252-3bcbd988-782e-485f-ae12-c8cb02e1439b.png)